packages/nimble-components/.storybook/blocks/StoryLayout.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
```

Comment:
Could be useful to add some docs for this infrastructure:
1. a short README in the `blocks` folder explaining what goes there
2. a JSDoc comment on each block (at least the top level ones) explaining what it's used for and maybe even a code snippet showing its usage